### PR TITLE
[34261] Add tax exemption category to invoice api and handle site default on …

### PR DESCRIPTION
**Pull request by ChristopherCotnoir**
…items

### CODE REVIEW

**Comment by ChristopherCotnoir** [comment]
Required by fixed-assets-commercial#74.

File: foundation-database/api/views/invoiceline.sql [hunk 1]
```diff
@@ -128,6 +130,7 @@ BEGIN
 		END,
 		getGlAccntId(pNew.alternate_rev_account),
 		COALESCE(pNew.invoice_subnumber,0)
```
Comment (andersdd) [inline foundation-database/api/views/invoiceline.sql]: Missing a comma on this line - it's the cause of the CI failure

**Comment by harithamekala** [comment]
Observed SQL Error while build app the database. Please referred the attached screenshot
3567_pullrequest

**Comment by andersdd** [comment]
I hit this problem with my earlier change.  You need to update the test as well to avoid this CI failure

 test/database/views/invoiceline.js

**Comment by harithamekala** [comment]
Performed UAT with Fixed Assets 1.8.5 package: Issue still exists. User is not able to perform ‘Sell Asset’ when both ‘Sales Amount’ and ‘Tax’ are assigned. Please refer to the attached screenshot ‘PullRequest_3567.png’.
pullrequest_3567

**Pull request merged**